catalog/malariaatlasproject/projects_malariaatlasproject_assets_accessibility_friction_surface_2015_v1_0.jsonnet

```diff
@@ -70,12 +70,12 @@ local license = spdx.cc_by_4_0;
   extent: ee.extent(-180.0, -60.0, 180.0, 85.0,
                     '2015-01-01T00:00:00Z', '2016-01-01T00:00:00Z'),
   summaries: {
+    gsd: [927.67],
     'eo:bands': [
       {
         name: 'friction',
         description: 'Land-based travel speed.',
-        'gee:units': 'minutes/meter',
-        gsd: 927.67,
+        'gee:units': 'min/m',
```

Comment:
Thanks for the change! Adding this string to checker/units.py should fix the test failures.